shared/global-errors/index.desktop.js

```diff
@@ -79,43 +80,71 @@ class GlobalError extends Component<void, Props, State> {
     }[size]
   }
 
-  render () {
+  renderReachability () {
+    return (
+      <Box style={{...containerOverlayStyle}}>
+        <Box style={{...overlayRowStyle}}>
+          <Text type='BodyBig' style={{color: globalColors.white, textAlign: 'center'}}>Keybase is currently unreachable. Trying to reconnect you…</Text>
+        </Box>
+        <Box style={{...overlayFillStyle}}>
+          <Icon type='icon-loader-connecting-112' />
+        </Box>
+      </Box>
+    )
+  }
+
+  renderError () {
     const {onDismiss} = this.props
     const summary = this.state.cachedSummary
     const details = this.state.cachedDetails
     const maxHeight = GlobalError.maxHeightForSize(this.state.size)
 
     return (
-      <Box style={{...containerStyle, maxHeight}} onClick={this._onExpandClick}>
-        <Box style={summaryRowStyle}>
+      <Box style={{...containerStyle, ...containerErrorStyle, maxHeight}} onClick={this._onExpandClick}>
+        <Box style={{...summaryRowStyle, ...summaryRowErrorStyle}}>
           {summary && <Icon type='iconfont-exclamation' style={{color: globalColors.white, marginRight: 8}} />}
-          <Text type='BodySmall' style={{color: globalColors.white, textAlign: 'center'}}>{summary}</Text>
+          <Text type='BodyBig' style={{color: globalColors.white, textAlign: 'center'}}>{summary}</Text>
           {summary && <Icon type='iconfont-close' onClick={onDismiss} style={{position: 'absolute', right: 8, color: globalColors.white_75}} />}
         </Box>
-        <Text type='BodySmall' style={detailStyle}>{details}</Text>
+        <Text type='BodyBig' style={detailStyle}>{details}</Text>
       </Box>
     )
   }
+
+  render () {
+    if (this.props.reachability && this.props.reachability.reachable === ReachabilityReachable.no) {
+      return this.renderReachability()
+    }
+    return this.renderError()
+  }
 }
 
 const containerStyle = {
   ...globalStyles.flexBoxColumn,
-  ...transition('max-height'),
   overflow: 'hidden',
   position: 'absolute',
   top: 0,
   left: 0,
   right: 0,
-  backgroundColor: globalColors.black_75,
+  zIndex: 1000,
```

Comment:
It turns out we have a bunch of zIndex hacks that this needs to be set to workaround them.
I created an issue to remove those here: https://keybase.atlassian.net/browse/DESKTOP-2348

I started to remove them and then realized it would be hard for me to test all these places to make sure I didn't break anything, so...

Did move the GlobalError though.